CONTRIBUTING.md

```diff
@@ -72,14 +71,9 @@ is no automatic restart mechanism. Alternatively, keep the developer tools open
 <kbd>Cmd</kbd> + <kbd>R</kbd> (macOS) or <kbd>Ctrl</kbd> + <kbd>R</kbd>
 (Windows & Linux).
 
-Also, note that the assets loaded by the application are not necessarily the same files
-you’re touching. You may not see your changes until you run `yarn grunt` on the
-command-line like you did during setup. You can make it easier on yourself by generating
-the latest built assets when you change a file. Run this in its own terminal instance
-while you make changes:
-
 ```
-yarn grunt dev # runs until you stop it, re-generating built assets on file changes
+yarn build-everything:watch # runs until you stop it, re-generating built assets on file changes
+# once this command is waiting for changes, you will need to run in another terminal `yarn parcel-util-worker` to fix the "exports undefined" error on start
```

Comment:
Probably need to mention that if you change the SASS you need to run yarn sass / yarn sass:watch separately

Reply:
updated